docs/building-cmake.md

```diff
@@ -1,3 +1,5 @@
+**NOTE: THIS INSTRUCTION IS STAYED HERE FOR HISTORICAL PURPOSES AND FOR WINDOWS XP USERS WHICH WANTS TO TRY TO COMPILE KEPKA ON WINDOWS XP. IN THAT CASE YOU SHOULD USE PARTS OF THIS DOC RELATED TO PATCHING Qt 5.6 AND IN ADDITIONAL YOU MUST USE XP MSVS TOOLHAINS!**
```

Comment:
I can't see how this document could help Windows XP users. It describes a Linux build.